_safari/Save to Pocket/Info.plist

```diff
@@ -28,5 +28,10 @@
 	<string>Main</string>
 	<key>NSPrincipalClass</key>
 	<string>NSApplication</string>
+	<key>UIAppFonts</key>
```

Comment:
This is an iOS thing. I don't think it has any effect in a macOS app. :)

It looks like this got clobbered somewhere along the line:

```	
<key>ATSApplicationFontsPath</key>
<string>fonts</string>
```

Reply:
Added this back in in place of the UIAppFonts